homeassistant/components/microsoft/tts.py

```diff
@@ -91,6 +102,7 @@
         vol.Optional(CONF_PITCH, default=DEFAULT_PITCH): cv.string,
         vol.Optional(CONF_CONTOUR, default=DEFAULT_CONTOUR): cv.string,
         vol.Optional(CONF_REGION, default=DEFAULT_REGION): cv.string,
+        vol.Optional(CONF_OUTPUT, default=DEFAULT_OUTPUT): vol.In(OUTPUTS),
```

Comment:
We no longer allow new integrations to use a platform config section or old integrations to do changes to the platform config section. See https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md.

Please refactor this integration to have the config schema in the base integration module and the config yaml section under the top level integration domain key.